frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditMyInfoChildren/EditMyInfoChildren.tsx

```diff
@@ -63,6 +64,14 @@ export const EditMyInfoChildren = ({
   return (
     <CreatePageDrawerContentContainer>
       <VStack align="flex-start">
+        <VStack>
```

Comment:
Seems like we have two layers of `<VStack />` here. Since the child element `<Box />` is the only child, we can safely omit the newly introduced `<VStack />`.